Frontend/ui-customer/src/Components/MessagingView/Controller.purs

```diff
@@ -19,13 +19,18 @@ import MerchantConfig.Types
 import MerchantConfig.DefaultConfig as DC
 import PrestoDOM (BottomSheetState(..))
 import Common.Resources.Constants (chatSuggestion)
+import Data.Maybe (Maybe(..))
+import Language.Strings (getString)
+import Language.Types (STR(..))
 
 data Action = SendMessage
             | SendSuggestion String
             | BackPressed
             | TextChanged String
             | Call
             | NoAction
+            | MultiChat
```

Comment:
can we rename this according to its functionality.

Reply:
Shall I rename it as MultiChatContactList?